en/snap/with-plugins.md

```diff
@@ -56,269 +60,298 @@ Here is the list of Content Management System plugins and extensions that are su
 
 <div class="my-card">
 
-#### [ Wordpress - Easy Digital Download](#wordpress-easy-digital-download)
+#### [ WordPress - Easy Digital Download](#wordpress-easy-digital-download)
 </div>
-<!-- TODO: explain what is notifcation URL for non dev user. Why it is important, and what's the benefit of using notification url -->
+<!-- TODO: explain what is notification URL for non dev user. Why it is important, and what's the benefit of using notification url -->
 <hr><br><br>
 
-### Wordpress - WooCommerce
-<hr>
-
-Midtrans  ❤️ WooCommerce! This plugin will allow secure online payment on your WooCommerce store, without your customer ever need to leave your WooCommerce store! With beautiful responsive payment interface built-in. We strive to make payments simple for both the merchant and customers. Support various online payment channel. Support WooCommerce v3 & v2.
+## WordPress - WooCommerce
+Midtrans ❤️ WooCommerce! This plugin allows secure online payment on your WooCommerce store, without ever needing your customer to leave your WooCommerce store! It has a beautiful built-in responsive payment interface. Midtrans strives to make payments simple for you and your customers. It supports various online payment channels. Midtrans supports WooCommerce v2 and v3.
 
-Our WooCommerce plugins also available on [Wordpress plugins store](https://wordpress.org/plugins/midtrans-woocommerce/). If it's not listed there, you can always download and [install as described below](#b-manual-installation).
+Midtrans-WooCommerce plugin is also available on [WordPress plugins store](https://wordpress.org/plugins/midtrans-woocommerce/). If you cannot find it listed there, you can always download and install it manually. For more details, refer to Manual Installation.
 
-#### Requirements:
-* WordPress v3.9 or greater **|** Tested up to v5.x
-* WooCommerce v2 or greater **|** Tested up to v3.5.2
-* PHP version v5.4 or greater
-* MySQL version v5.0 or greater
+#### Requirements
+Some of the requirements to continue with the integration process, are listed below.
+* WordPress v3.9 or later **|** Tested up to v5.x
+* WooCommerce v2 or later **|** Tested up to v3.5.2
+* PHP version v5.4 or later
+* MySQL version v5.0 or later
 * PHP CURL enabled server/host
+* Midtrans plugin for WooCommerce [ [GitHub](https://github.com/veritrans/SNAP-Woocommerce) | [Zip](https://github.com/veritrans/SNAP-Woocommerce/archive/master.zip) ]
```

Comment:
I think we should improve this to be more intuitive for reader. Please change this part of download requirement (applies to all CMS) into:

> * Download Midtrans plugin for XYZ: [Zip file](https://example.org) (Open sourced on [GitHub](https://example.org))

And for those with multiple links:

> * Download Midtrans plugin for XYZ (please choose accordingly): 
>   * For XYZ v1.x: [Zip file](https://example.org) (Open sourced on [GitHub](https://example.org))
>   * For XYZ v2.x: [Zip file](https://example.org) (Open sourced on [GitHub](https://example.org))

Thanks

Reply:
Ok. I will do that